packages/mongodb-rag-ingest/src/sources/LangchainDocumentLoaderDataSource.ts

```diff
@@ -0,0 +1,62 @@
+import { DocumentLoader } from "langchain/dist/document_loaders/base";
+import { Document as LangchainDocument } from "langchain/document";
+import { DataSource } from "./DataSource";
+import { Page, PageMetadata, PageStore } from "mongodb-rag-core";
+
+export interface MakeLangChainDocumentLoaderDataSourceParams {
+  /**
+    [Langchain document loader](https://js.langchain.com/docs/modules/data_connection/document_loaders/) to use to load documents.
+   */
+  documentLoader: DocumentLoader;
+
+  /**
+    Name of the data source used by MongoDB RAG Ingest.
+   */
+  name: string;
+
+  /**
+    Metadata to use in the page metadata of all documents.
+
+    `Page.metadata` generated with `transformLangchainDocumentToPage()`
+    overrides this metadata if the properties have the same key.
+   */
+  metadata?: PageMetadata;
+
+  /**
+    Take the {@link LangchainDocument} returned by the `documentLoader`
+    and transform it into the {@link Page} persisted in the {@link PageStore}.
```

Comment:
```suggestion
    and transform it into the {@link Page} that can be persisted in the {@link PageStore}.
```

This method doesn't actually store the page right?